### ISSUE TITLE
Config file for exchanges

### ISSUE THREAD

**Issue by RatkoR** [askmike/gekko#746]
A question about an [exchanges.js](https://github.com/askmike/gekko/blob/stable/exchanges.js) file.

Would it make sense if every exchange in the `/exchanges` dir has it's info there? For example, the [poloniex](https://github.com/askmike/gekko/blob/stable/exchanges/poloniex.js) exchange file would have it's capabilities listed in it's .js file.

Currently, exchange data is located in two places (in `exchanges.js` and in `exchanges/poloniex.js`). Adding a new exchange is troublesome, because it's not enough to just add a new file in the exchanges dir, you have to change the exchanges.js file. Which is bad, because this file is managed by the upstream branch. What I want to say is that one cannot just drop a new exchange in the `exchanges/` dir and play with it.

In case of poloniex.js (and every other exchange), it could have a `capabiliteis()` method with the exact same info as is now in exchanges.js:

``` 
Trader.prototype.capabilities = function() {
    return {
        currencies: ['BTC', 'ETH', 'XMR', 'USDT'],
	assets: [
          '1CR', 'ABY', 'AC', 'ACH', 'ADN', 'AEON', 'AERO', 'AIR', 'AMP', 'APH', ...
        ],
        markets: [
          { pair: ['BTC', 'YC'], minimalOrder: { amount: 0.0001, unit: 'asset' } },
          ....
        ],
        requires: ['key', 'secret'],
        tid: 'tid',
        providesHistory: 'date',
        providesFullHistory: true
    }
}
```

**Comment by askmike** [comment]
Interesting idea! I like the exchanges.js file a lot since you can see and compare (and copy) different exchanges at a glance, but it's getting out of hand with the gazillion markets supported for Poloniex.

But if we are going for this it would require changing this for all supported exchanges (not something I feel like doing). Also note that: the UI imports this file to render the interface, if you are going to split it up, also provide a way to retrieve this data from the frontend (API call I guess).

Talking about the UI:

> Adding a new exchange is troublesome, because it's not enough to just add a new file in the exchanges dir, you have to change the exchanges.js file.

You also need to recompile the frontend, so if you want this to be more like "drop a file into `/exchanges` and it works" make it fully dynamic so the frontend redraws the UI based on files in this folder (or possibly on refresh).

**Comment by askmike** [comment]
discussion moved to #750.
